docker-compose.yml

```diff
@@ -13,10 +13,12 @@ services:
         - MYSQL_DATABASE=chanjo4_test
         - MYSQL_USER=chanjoUser
         - MYSQL_PASSWORD=chanjoPassword
-      healthcheck: # Wait for the service to be ready before accepting incoming connections
-        test: "mysql --user=chanjoUser --password=chanjoPassword --execute \"SHOW DATABASES;\""
-        timeout: 10s
-        retries: 20
+      healthcheck:
+          test: [ "CMD", "healthcheck.sh", "--connect", "--innodb_initialized" ]
+          start_period: 10s
+          interval: 10s
+          timeout: 5s
+          retries: 3
       networks:
```

Comment:
After this fix:

<img width="1344" alt="image" src="https://github.com/user-attachments/assets/bb00fbe1-6020-43e9-9a08-efa4099aa322">

Reply:
I just copied the official healthcheck step from here: https://mariadb.com/kb/en/using-healthcheck-sh/